crates/astria-sequencer/src/upgrades/upgrades_handler.rs

```diff
@@ -58,7 +58,8 @@ impl UpgradesHandler {
         upgrades_filepath: P,
         cometbft_rpc_addr: String,
     ) -> Result<Self> {
-        let upgrades = Upgrades::new(upgrades_filepath).wrap_err("failed constructing upgrades")?;
+        let upgrades =
+            Upgrades::read_from_path(upgrades_filepath).wrap_err("failed constructing upgrades")?;
```

Comment:
report the path? i.e. `wrap_err_with(|| format!("failed constructing upgrades from file at `{}`", upgrades_filepath.as_ref().display())`?

although I am also fine if the assumption is that the caller of` UpgradesHandler::new` should report this.